hummingbot/connector/exchange/ascend_ex/ascend_ex_auth.py

```diff
@@ -13,13 +14,37 @@ def __init__(self, api_key: str, secret_key: str):
         self.api_key = api_key
         self.secret_key = secret_key
 
+    def get_headers(self,
+                    path_url: str = None,
+                    data: Dict[str, Any] = None) -> Dict[str, Any]:
+        """
+        Generates final headers
+        :param path_url: URL of the auth API endpoint
+        :param data: data to be included in the headers
+        :return: a dictionary of request info including auth & meta headers
+        """
+        if path_url is not None:
+            headers = {
+                **self.get_meta_headers(),
+                **self.get_auth_headers(path_url, data),
+                **self.get_hb_id_headers(),
+            }
+        else:
+            headers = {
+                **self.get_meta_headers(),
+                **self.get_hb_id_headers(),
+            }
+        return headers
+
```

Comment:
It should be possible to populate the non-auth headers in a pre processor configured in the web assistant